### PR TITLE
Add nvidia support for ubuntu16 ppcle

### PR DESCRIPTION
**Pull request by jdekonin**
* eclipse/openj9#7432

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml [hunk 1]
```diff
@@ -102,3 +102,37 @@
...
+    dest: /tmp/cuda-toolkit-9-0_9.0.176-1_ppc64el.deb
+  when:
+    - cuda_installed.stat.isdir is not defined
+    - (ansible_distribution == "Ubuntu" and ansible_distribution_major_version == "16")
+    - ansible_architecture == "ppc64le"
+  tags: nvidia_cuda_toolkit
+
+- name: Add an Apt signing key for Ubuntu 16 nvidia cuda repo
+  apt_key:
+    url: http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/ppc64el/7fa2af80.pub
```
Comment (sxa555) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: Can we pull this over https instead?

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml [hunk 2]
```diff
@@ -102,3 +102,37 @@
     - nvidia_cuda_toolkit
     # TODO: Package installs should not use latest
     - skip_ansible_lint
+
+- name: Download NVidia CUDA Repo RPM - Ubuntu 16 on PPC64LE
```
Comment (sxa555) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: Does this need to be locked to Ubuntu 16? i.e. is there a reason not to install on any Ubuntu 18.04 systems?

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml (hunk 1 above)
Comment (jdekonin) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: Yes, change made.

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml (hunk 2 above)
Comment (jdekonin) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: The nvidia repo url is o/s version and platform specific, same as the key; `ubuntu1604/ppc64el`

**Comment by jdekonin** [comment]
I believe this is good now.

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml (hunk 2 above)
Comment (sxa555) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml]: Disappointing :'(

**Pull request merged**